tools/clang/test/SemaHLSL/hlsl/objects/texture/CalculateLODExtraDiag.hlsl

```diff
@@ -2,7 +2,7 @@
 
 // Check cs/as/mesh and node.
 
-SamplerState ss : register(s2);
+SamplerComparisonState ss : register(s2);
```

Comment:
I think this change was made to make these diagnostics show up in the earlier understanding of the pseudo-code. I think this can stay a SamplerState, though perhaps we should have both and mix up uses a bit.

Reply:
Per above, this is no longer relevant.